### ISSUE TITLE
Compare Papermerge with other projects

### ISSUE THREAD

**Issue by ARandomOWL** [ciur/papermerge#25]
How does Papermerge compare with other projects such as MayanEDMS and Paperless? This would be a good addition to the readme.

**Comment by ciur** [comment]
Good one!
Comparison with Paperless would be easy to add because I know very well the project.
Internal refactoring for version 1.3.0 was heavily inspired by Paperless. I simplified many aspects of Papermerge to be similar to Paperless. For example Paperless used sqlite3 by default - so did I for Papermerge version 1.3.0. Another example: Paperless used a built-in consumer - the one you start with command:
 ` ./manage.py consume`

So did I - Papermerge now (> 1.3.0) has a built-in consumer - which I name worker - for me is more intuitive term:
   `./manage.py worker`.

In short, if you ever used Paperless - you will definitely like Papermerge. 

Paperless is a fantastic project - I really love it and its author is a very talented developer - as a result code for Paperless is very clean and well written. Unfortunately it is not actively developed anymore. This is one of the biggest difference - Papermerge is actively developed.

I don't know MayanEDMS very well, so it will take some time to learn little bit about it.
In any case, adding a comparison with other two leading FOSS DMS is a good idea!

**Comment by ciur** [comment]
[Included](https://github.com/ciur/papermerge#comparison-with-other-dms). I would be glad to learn other comparison matrics.
